src/datepicker-input/datepicker-input.component.ts

```diff
@@ -78,7 +78,7 @@ export class DatePickerInput {
 
 	@Output() valueChange: EventEmitter<string> = new EventEmitter();
 
-	@Output() toggleCalendar: EventEmitter<boolean> = new EventEmitter();
+	@Output() selected = new EventEmitter();
```

Comment:
Not sure that the semantics of `selected` as used elsewhere in the codebase make sense here as well ... there's no value emitted for one thing

Reply:
Would something like openCalendar be better?

Reply:
You may even just be able to forward the `click` event - that way all the cases are covered

Reply:
Otherwise `open` or `openCalendar` would probably make more sense ... but you'd want to make sure they fire for all cases of clicking on the input

Reply:
Okay, that makes sense